files/virus_scan.sh

```diff
@@ -7,7 +7,12 @@ LAST_SCAN_LOG_FILENAME='/var/log/clamav/lastscan.log'
 LAST_DETECTION_FILENAME='/var/log/clamav/last_detection'
 
 # scan the entire system and write to the log
-clamdscan --multiscan --infected --fdpass / &> ${LAST_SCAN_LOG_FILENAME}
+clamscan --infected --recursive \
+         --log=${LAST_SCAN_LOG_FILENAME} \
+         --exclude-dir=/dev \
+         --exclude-dir=/sys \
+         --exclude-dir=/proc \
+         /
```

Comment:
Why are we now excluding these directories from the scanning?  The comment says "scan the entire system" and this is no longer doing that.

Reply:
Those directories were being excluded via the ClamAV daemon config file before I made any changes.  I had to add those lines to the script to exclude them now that we are using `clamscan` instead of `clamdscan`.

I agree with the code then and now; I don't think it makes sense to scan these particular parts of the root filesystem.  I am happy to change the comment, or remove the exclusions if the group really wants that.

What say you, @felddy and @mcdonnnj?

Reply:
@dav3r, see 0c49430 for a more accurate comment.

Reply:
I'm good if that was the original behavior, but let's hear from @felddy and @mcdonnnj too.  The updated comment is helpful.

Reply:
That's the previous behavior.  See the ansible configuration block that was removed below. 
This is good.